src/dndbeyond/content-scripts/encounter.js

```diff
@@ -26,20 +26,24 @@ function documentModified(mutations, observer) {
 }
 
 function updateCombatTracker() {
-    if ($(".turn-controls__next-turn-button").length) {
-        $(".ui-dialog-buttonpane").hide(); // get rid of turn controls, to make it more obvious to do things from DDB.
-        const combat = Array.from($(".combatant-card.in-combat")).map(x => [
-            $(".combatant-summary__name", x).text(),
-            $(".combatant-card__initiative-value", x).text(),
-            $(x).hasClass("is-turn"),
-        ]);
-        const req = {
-            action: "combat-tracker",
-            combat,
-        };
-        console.log("Sending combat update", combat);
-        chrome.runtime.sendMessage(req, resp => beyond20SendMessageFailure(character, resp));
+    if (!$(".turn-controls__next-turn-button").length) {
+        return;
     }
+    $(".ui-dialog-buttonpane").hide(); // get rid of turn controls, to make it more obvious to do things from DDB.
```

Comment:
oh yeah, right, I wanted to check this, what are you hiding here? From the comment it sounds like you're hiding controls from roll20, but that wouldn't work since this is the encounter's page content script. Also, I would not hide any UI element since there's no way to unhide it

Reply:
Yeah I realised that when I was doing the other changes, and have since removed it. I'm still on the fence as to whether we should be hiding them, especially without a way to bring them back should they decide to stop using the tracker mid-combat.

Reply:
Yeah, I'd definitely say not to hide it. I don't see a point in it, and it could only cause problems (due to the software) rather than eliminate them (due to the user). I prefer that if there is a problem, it's the user's fault rather than the software's :p